mod_modular_vanilla/hooks/skills/skill.nut

```diff
@@ -586,8 +586,15 @@
 			}
 		}
 	}
-});
 
+	// MV: Added
+	// Can be used to modify the result of behavior.queryTargetValue
+	// Must return the new value
+	q.getQueryTargetValueMult <- function( _entity, _target, _skill )
```

Comment:
Do you think it is worthwhile to also have the `behavior` as an optional parameter in this function? A skill may want to apply a stronger target value mult for a certain behavior compared to another one. Or is that not necessary?